### DISCUSSION TITLE
[Help]: Wheel gestures

### DISCUSSION THREAD

**Discussion by arashagp** [davidjerleke/embla-carousel#1013] [Help] [answered]
### Summary

Hello,

I am using your carousel plugin and would like to request some guidance on handling scroll behavior. Specifically, I want to achieve the following:

When I reach the last slide of the carousel, instead of allowing the carousel section to scroll, I would like the page (window) to continue scrolling. 

Here’s a brief overview of my implementation:
- I am using the wheel gestures in a section of my page.
- When the user scrolls down and reaches the last index of the carousel, I want to disable scrolling within that carousel section and allow the main window to scroll instead.

Could you please provide me with guidance on how to implement this behavior? Any examples or code snippets would be greatly appreciated.

Thank you for your help!

Best regards,
arashagp

### If applicable, which variants of Embla Carousel are relevant to this question?

- [X] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-fade
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

**Comment by njfamirm** [comment]
video capture of the issue:

![CleanShot 2024-09-23 at 13 37 28](https://github.com/user-attachments/assets/96c797d2-2a48-42d0-84c2-41286943422c)

**Reply by davidjerleke** [reply, marked as answer]
Hi @arashagp,

Wrong repository, the Wheel gestures plugin is maintained separately. Please create an issue [here](https://github.com/xiel/embla-carousel-wheel-gestures/issues) instead.
